lib/libxdp/Makefile

```diff
@@ -9,7 +9,7 @@ include $(LIB_DIR)/util/util.mk
 OBJDIR ?= .
 SHARED_OBJDIR := $(OBJDIR)/sharedobjs
 STATIC_OBJDIR := $(OBJDIR)/staticobjs
-OBJS := libxdp.o
+OBJS := libxdp.o xsk.o
```

Comment:
Please make sure the new header files are also added as dependencies so they cause recompiles. At least libxdp_internal.h is not right now...